source/account_structure/set_up_where_services_link_to/index.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: Set up how your services link to GOV.UK Pay services
+last_reviewed_on: 2021-02-10
+review_in: 6 months
+weight: 142
+---
+
+# Set up how your services link to GOV.UK Pay services
+
+Each GOV.UK Pay service can take payments from one or more of your services.
+
+GOV.UK Pay services are free, and there's no limit to how many you can create.
+
+## Link your services to a single GOV.UK Pay service
+
+You can use one GOV.UK Pay service to take payments for a group of your services, if you need to easily manage the following across the whole group:
+
+- your staff members' access in the GOV.UK Pay admin tool
+- your account settings, for example which card types you accept
+
+1. [Create one GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create).
+2. Use API keys or payment links from the GOV.UK Pay service in all of the services in your group.
+
+You will not be able to:
+
+- give your services different names - but you can still show your users a different [description](/making_payments/#description) for each service
+- send payments to different bank accounts
+
+If you need to know which service each payment came from when you [report on payments](/reporting), you can:
+
+- add different [reference numbers](/making_payments/#reference)
+- add different [descriptions](/making_payments/#description)
+- use [custom metadata](/custom_metadata/#add-custom-metadata)
+- use different [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
+
+## Link your services to different GOV.UK Pay services
+
+You should [create a separate GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create) for each of your services if you need to do any of the following:
```

Comment:
i wonder if we do need to add a link to MOTO in here. def not essential but I wonder if some people might come here before investigating the MOTO page?